src/main/commandServer/httpCommandServer.ts

```diff
@@ -39,8 +39,9 @@ export class HttpCommandServer {
     v0: {
       GET: { settings: this.listSettings },
       PUT: {
-        shutdown: this.wrapShutdown,
-        settings: this.updateSettings
+        factoryReset: this.factoryReset,
```

Comment:
Bike shedding: `/v0/factory-reset`? (I'm not very set, though, just bringing it up because having upper case in URLs is not as common).

Though of course that means extra effort on the JS side…

Reply:
Good point. The CF API uses all_lower_case names, so let's stick with that.